cmd/pbm/status.go

```diff
@@ -28,6 +32,172 @@ import (
 	"github.com/percona/percona-backup-mongodb/sdk/cli"
 )
 
+type diagnosticOptions struct {
```

Comment:
Proposing to extract all diagnostic logic into separate file (e.g. diagnostic.go).